### ISSUE TITLE
Frameworks are not access using screen reader

### ISSUE THREAD

**Issue by jmarks** [opensalt/opensalt#236]
Can not navigate to a framework from the home page (cfdoc) or within a tree view using tab and enter. You can navigate to the header and footer options, and to the item detail within a framework. All elements that can be clicked on on the screen should be able to highlighted using the table key, and selected using the enter key.

**Comment by roverwolf** [comment]
What screen reader were you using?

I'll agree that this could be a defect, but it is not a bug as there have been no requirements around this.

Note that you need to use the arrow keys within the tree to navigate.

On the home page there does seem to be an issue as the tree does not highlight upon focus and one can not select any but the first framework from each creator folder using keyboard navigation.

Being able to focus the tree using tab and navigate it using arrow keys does work when viewing a framework.

**Comment by brandonopened** [comment]
I would like to talk about prioritizing this this week as well. I've gotten some pushback too about lack of accessibility features.
